Makefile

```diff
@@ -29,10 +29,13 @@ checkgofmt:
 vet:
 	go vet ./...
 
+# Check ST1005 checks that error strings are not capitalized
+# We want to allow this
+
 .PHONY: staticcheck
 staticcheck:
 	@go get honnef.co/go/tools/cmd/staticcheck
-	staticcheck ./...
+	staticcheck -checks 'all -ST1005' ./...
```

Comment:
:/

Maybe add a comment to disable this just in the violating file?

Reply:
https://staticcheck.io/docs/#ignoring-problems

Reply:
Yea I'll figure out a nicer way to do this that just ignores that one file.

Reply:
Done